src/main.js

```diff
@@ -51,39 +58,48 @@ class WebdriverPercy {
     browser.percy.environment = new Environment(process.env);
     browser.percy.percyClient = new PercyClient({ token, apiUrl, clientInfo });
 
-    browser.addCommand('__percyReinit', function async() { // eslint-disable-line prefer-arrow-callback
+    // adding `async` as function name disables the synchronous behavior of WebdriverIO commands
+    // eslint-disable-next-line prefer-arrow-callback
```

Comment:
The async function name is given special treatment by wdio, so make sure we can include it here.